index.js

```diff
@@ -238,7 +238,7 @@ function initTimer () {
     displayOutput.textContent = displayString
     update(timeLeft, wholeTime)
   }
-  pauseBtn.onclick = (e) => { pauseTimer() }
+  pauseBtn.addEventListener('click', (e) => { pauseTimer() })
```

Comment:
unfortunately this is causing some problems (like pause not working after odd-numbered restarts) because this adds a new event listener every time a timer is started. will look into a fix.

Reply:
ahh, yeah, the event listeners need to be removed then before being added.